packages/front-end/components/Tags/TagsInput.tsx

```diff
@@ -87,25 +84,35 @@ const TagsInput: FC<{
         },
       };
     },
+    control: (styles, { isFocused }) => {
+      return {
+        ...styles,
+        boxShadow: `0px 0px 0px 1px ${isFocused ? "var(--violet-8)" : "000"}`,
+      };
+    },
```

Comment:
Need to remove this as I'm handling this in a separate PR [here](https://github.com/growthbook/growthbook/pull/3509).